crates/core/src/db/db_metrics/mod.rs

```diff
@@ -75,6 +75,16 @@ metrics_group!(
         #[help = "The cumulative number of rolled back transactions"]
         #[labels(txn_type: TransactionType, db: Address, reducer: str)]
         pub rdb_num_txns_rolledback: IntCounterVec,
+
+        #[name = spacetime_txn_elapsed_time_us]
+        #[help = "The total elapsed (wall) time of a transaction (microseconds)"]
```

Comment:
As transactions get faster, I do wonder if we will wish this was nanoseconds. Curious what others think.